theme/less/_navigation.less

```diff
@@ -1,26 +1,70 @@
-nav
+header nav
 {
-  ul
+  border-top: 1px solid #E9E9E9;
+  background: #FFFFFF;
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+}
+
+header nav ul
+{
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+header nav ul a
+{
+  display: block;
+  font-size: 24px;
+  text-decoration: none;
+  color: #626262;
+  padding: 25px;
+}
+
+#hamburger-menu
+{
+  text-align: center; 
+}
+
+#hamburger-menu ul
+{
+  display: block;
+}
+
+#hamburger-menu a
+{
+  padding: 10px;
+  font-size: 13px;
```

Comment:
Making fonts that small can affect accessiblity on mobile. Make a bit bigger? Or reuse same size?

Reply:
You mean reusing 24px? then we can actually not to define additional `font-size` here.

Reply:
I haven't tested for optimal size, if `24px` doesn't look too bulky/bad then please reuse 24px.

> then we can actually not to define additional

Sure if 24px looks fine, then you can just remove this line.

Reply:
Done.